packages/server/src/admin/admin.module.ts

```diff
@@ -0,0 +1,29 @@
+import { Module } from '@nestjs/common';
+import { AdminCoreModuleFactory, AdminAuthModuleFactory, DefaultAdminSite } from 'nestjs-admin';
+import { adminCredentialValidator } from './credentialValidator';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { AdminUserModel } from './admin-user.entity';
+import { CourseAdmin, QueueAdmin, UserAdmin } from './admin-entities';
+import { UserModel } from 'profile/user.entity';
+import { QueueModel } from 'queue/queue.entity';
+
+const CoreModule = AdminCoreModuleFactory.createAdminCoreModule({});
+const AuthModule = AdminAuthModuleFactory.createAdminAuthModule({
+  adminCoreModule: CoreModule,
+  credentialValidator: adminCredentialValidator, // how do you validate credentials
+  imports: [TypeOrmModule.forFeature([AdminUserModel])], // what modules export the dependencies of the credentialValidator available
+  providers: [],
+});
+
+@Module({
+  imports: [CoreModule, AuthModule],
+  exports: [CoreModule, AuthModule],
+})
+export class AdminModule {
+
+  constructor(private readonly adminSite: DefaultAdminSite) {
+    adminSite.register('Course', CourseAdmin);
```

Comment:
here's where we could register more entities. I didn't think it was worth adding panels for questions etc.

Reply:
We might want to add OfficeHour entities here for testing